src/components/ValidatorForm.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { ResourceForm } from './ResourceForm';
+
+import { SelectOption } from '../models/SelectOption';
+import { ProfileForm } from './ProfileForm';
+
+export interface ValidatorProps {
```

Comment:
It doesn't look like this interface is used anywhere else so I don't think it needs to be exported.